projects/1.20/assets/modrinth-deepdrilling/deepdrilling/lang/zh_cn.json

```
@@ -0,0 +1,63 @@
+{
+  "block.deepdrilling.andesite_drill_head": "安山岩制钻头",
+  "block.deepdrilling.asurine_node": "皓蓝石矿脉",
+  "block.deepdrilling.asurine_node.tooltip.behaviour1": "_皓蓝石，凝灰岩，方解石_",
+  "block.deepdrilling.asurine_node.tooltip.condition1": "伴生产物",
+  "block.deepdrilling.asurine_node.tooltip.summary": "在_寒冷生物群系_的_基岩层_中生成",
+  "block.deepdrilling.blank_module": "空白模块",
+  "block.deepdrilling.blank_module.tooltip.behaviour1": "_没什么用_... 但可以用来_连接_其他模块。",
```

```suggestion
  "block.deepdrilling.blank_module.tooltip.behaviour1": "_没什么用_……但可以用来_连接_其他模块。",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

需要实际检查游戏内是否存在显示问题，如果存在则需要在省略号后加空格或者使用三个英文句点代替